app/es_api/utils/queries.py

```diff
@@ -3,7 +3,8 @@
 import logging
 
 from configurations.models import CourseInformationMapping, XDSConfiguration
-from core.models import CourseSpotlight, SearchFilter, SearchSortOption
+from core.models import CourseSpotlight, SearchFilter
+from core.models import SearchSortOption, SearchField
```

Comment:
@mdonlon I am surprised lint didn't catch this, but these should be part of the same import statement, even if you have to split it across lines.  see `core/admin.py` if you need an example

Reply:
lint said that the line was too long so I split it, I fixed this now and will push an update.